Sources/Subs-Calendar.php

```diff
@@ -699,15 +697,25 @@ function getCalendarList($start_date, $end_date, $calendarOptions)
 	// Give birthdays and holidays a friendly format, without the year
 	$date_format = str_replace(array('%Y', '%y', '%G', '%g', '%C', '%c', '%D'), array('', '', '', '', '', '%b %d', '%m/%d'), get_date_or_time_format('date'));
 
+	// timeformat() uses $user_info to get the format
+	if (isset($user_info['time_format']))
+		$saved_dt_format = $user_info['time_format'];
+	$user_info['time_format'] = $date_format;
```

Comment:
This part is unnecessary. Although this behaviour is poorly documented, the `$show_today` parameter of `timeformat()` can either accept a boolean or a string containing a `strftime()` format string. There's no need to juggling around the existing value of `$user_info['time_format']`.

Reply:
Done.